src/reliability.jl

```diff
@@ -0,0 +1,78 @@
+using LinearAlgebra: isposdef
+
+struct Reliability{T <: Real}
+    alpha::T
+    dropped::Vector{Pair{Int, T}}
+end
+
+function Base.show(io::IO, x::Reliability)
+    println(io, "Reliability for all items: $(round(x.alpha; digits=4))")
+    isempty(x.dropped) && return
+    println(io, "")
+    println(io, "Reliability if an item is dropped:")
+    for i ∈ x.dropped
+        println(io, "item $(i.first): $(round(i.second; digits=4))")
+    end
+end
+
+"""
+    crombach_alpha(covmatrix::AbstractMatrix{<:Real})
+
+Calculate Crombach's alpha (1951) from a covariance matrix `covmatrix` according to
+the Wikipedia formula (https://en.wikipedia.org/wiki/Cronbach%27s_alpha):
```

Comment:
```suggestion
the [formula](https://en.wikipedia.org/wiki/Cronbach%27s_alpha):
```
(Wikipedia isn't an authoritative nor stable source)